pkgs/by-name/qd/qdl/package.nix

```
@@ -11,13 +11,13 @@
 
 stdenv.mkDerivation {
   pname = "qdl";
-  version = "unstable-2024-06-10";
+  version = "unstable-2025-19-03";
```

```suggestion
  version = "0-unstable-2025-03-19";
```
This is month-day, and versions must now start with a digit

---

Fixed also the commit message.